### PR TITLE
add news item about failing g.extension

### PR DESCRIPTION
**Pull request by veroandreo**
As suggested in the mailing list, I have created a news item to inform that installing extensions is currently failing. 

image

Please, review :)

### CODE REVIEW

**Comment by ninsbl** [comment]
MS Windows is likely not equally affected as addons are fetched as precompiled scripts there. Also, it is still possible to compile addons on UNIX like systems...

**Comment by veroandreo** [comment]
> MS Windows is likely not equally affected as addons are fetched as precompiled scripts there. Also, it is still possible to compile addons on UNIX like systems...

If you would please suggest the needed changes to the news text, I'd really appreciate it.

**Comment by ninsbl** [comment]
OK, I had to check the status on MS Windows. Here is a suggestion (for some reason I could not add it as a suggestion in the PR):

```
Due to a [re-structuring](https://github.com/OSGeo/grass-addons/issues/528) 
of the [addon repository](https://github.com/OSGeo/grass-addons),
the installation of addons with `g.extension' is currently not working
on UNIX-like systems.
Installation of addons with the current stable version on [OSGeo4W (Version 1)]
(https://download.osgeo.org/osgeo4w/osgeo4w-setup-x86-v1.exe)
is not affected.

All the necessary changes in 
[g.extension](https://grass.osgeo.org/grass-stable/manuals/g.extension.html) 
have been implemented, merged and back-ported.
The functionality will be back to normal very soon with 
the upcoming GRASS 7.8.6 and 8.0.0 releases.

In the meantime, you have two options: 
 1. [compile and install](https://grasswiki.osgeo.org/wiki/Compile_and_Install#Addons)
the addons manually after fetching the addon source code from [github](https://github.com/OSGeo/grass-addons).
 2. [compile and install](https://grasswiki.osgeo.org/wiki/Compile_and_Install) 
GRASS GIS from source, either `releasebranch_7_8` or `master` where 
g.extension changes have been applied

For the case of python addons, just download and run them as regular python
scripts.

Thanks for your patience! 

```

The [GRASS AddOns wiki](https://grasswiki.osgeo.org/wiki/Compile_and_Install#Addons) should probably be updated at some point...

**Comment by veroandreo** [comment]
Thanks, @ninsbl !! If you agree then, I'll merge soon.

**Review by ninsbl — APPROVED** [review]

File: content/news/2021_08_01_g_extension_currently_not_working.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Installing GRASS GIS extensions currently failing on some systems"
+date: 2021-08-01T00:00:00+00:00
+layout: "news"
+author: GRASS Development Team
+---
+
+Due to a [re-structuring](https://github.com/OSGeo/grass-addons/issues/528)
```
Comment (neteler) [inline content/news/2021_08_01_g_extension_currently_not_working.md:8]: Suggestion, since it didn't come by surprise:

Due to a long-term planned ...

**Review by neteler — APPROVED** [review]
One comment added

**Pull request merged**